src/components/atoms/TitleBar/TitleBar.styled.tsx

```diff
@@ -2,7 +2,6 @@ import styled from 'styled-components';
 
 import {DEFAULT_PANE_TITLE_HEIGHT} from '@constants/constants';
```

Comment:
Is this needed anymore?

Reply:
Yes it is used in TitleBar component

Reply:
Updated design again but on right bottom there should be two buttons the one button is a mistake on figma we confirmed that before we start this task 👍